ethexe/consensus/src/validator/coordinator.rs

```diff
@@ -49,14 +45,14 @@ impl StateHandler for Coordinator {
         &mut self.ctx
     }
 
-    fn into_context(self: Box<Self>) -> ValidatorContext {
+    fn into_context(self) -> ValidatorContext {
```

Comment:
we can do `trait StateHandler: Into<ValidatorContext>`

Reply:
`state.into()` will be ambiguous then